configs/application/UIComponents.json

```diff
@@ -595,7 +595,7 @@
 				"docked": "top",
 				"canGroup": false,
 				"options": {
-					"autoShow": false,
+					"autoShow": true,
```

Comment:
@zona-ciq why did you change this again? It should probably be in the PR description.

Reply:
I just tested with workspaceService.global set to true and toolbar did come up initially blank. I'll change it back for now. When we document the `global` attribute we need to be aware that if the `autoShow` field is set to "false" the toolbar wouldn't automatically show up on startup.